DemoPage/examples/PageContent/index.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+import React from 'react'
+import PageContent from '../../../layout/PageContent'
+import AsyncAction from '../../../buttons/AsyncActionToggle'
+import Button from '../../../buttons/Button'
+import LeadCopy from '../../../typography/LeadCopy'
+import sync from '../../../lib/sync'
+
+export default React.createClass({
+  displayName: 'PageContentExample',
+
+  unsubscribe(bool) {
+    let api = 'https://everydayhero-staging.com/api/v2/notification_options'
+    let namespace = 'page_received_donation'
+    let token = '62d35e46ee366be584e7d6529dd22077c021677d2efe21f03739dd30b39d3723'
+    let url = `${ api }/${ namespace }?access_token=${ token }&value=${ bool }`
```

Comment:
What's your opinion on using `const`'s instead of `let`'s here? I've been sort of defaulting to `const` and only using `let` when I intend for things to be redefined. Not sure if that's the way to go just curious